README.md

```diff
@@ -48,53 +44,58 @@ $ java -javaagent:./splunk-otel-javaagent.jar -Dotel.resource.attributes=service
     -jar target/java-agent-example-1.0-SNAPSHOT-shaded.jar https://google.com
 ```
 
-> :information_source: The `-javaagent` needs to be run before the `-jar` file,
+> The `-javaagent` needs to be run before the `-jar` file,
 > adding it as a JVM option, not as an application argument. For more
 > information, see the [Oracle
 > documentation](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/java.html).
 
+> :warning: Specify the agent as the only JVM agent for your application.
+> Multiple agents may result in unpredictable results, broken instrumentation,
+> and in some cases might crash your application.
+
+To see the Java Agent in action with sample applications, see our
+[examples](https://github.com/signalfx/tracing-examples/tree/master/opentelemetry-tracing/opentelemetry-java-tracing).
+
+### Basic Configuration
+
 The service name resource attribute is the only configuration option
 that typically needs to be specified. You can set it by adding a `service.name`
-attribute to the [OpenTelemetry Resource](https://github.com/open-telemetry/opentelemetry-java-instrumentation#opentelemetry-resource).
+attribute as shown in the [example above](#getting-started).
 
-A couple other configuration options that may need to be changed or
-set are:
+A couple other configuration options that may need to be changed or set are:
 
-* Endpoint if not sending to a locally running Smart Agent with default
-  configuration
-* Environment resource attribute (example:
+- Endpoint if not sending to a locally running Smart Agent with default
+  configuration. See the [Jaeger exporter](#jaeger-exporter) section for more information.
+- Environment resource attribute (example:
   `-Dotel.resource.attributes=service.name=my-service,environment=production`) to specify what
   environment the span originated from.
 
-The agent instruments supported libraries and frameworks with bytecode
-manipulation and configures an OpenTelemetry-compatible tracer to capture
-and export trace spans. The agent also registers an OpenTelemetry `getTracer`
-so you can support existing custom instrumentation or add custom
-instrumentation to your application later.
+### Supported Java Versions
 
-To see the Java Agent in action with sample applications, see our
-[examples](https://github.com/signalfx/tracing-examples/tree/master/opentelemetry-tracing/opentelemetry-java-tracing).
+The agent works with Java runtimes version 8 and higher and supports all
+JVM-based languages (for example, Clojure, Groovy, Kotlin, Scala). Supported
```

Comment:
Out of curiousity, has anyone tried using otel with Clojure? I'd love to see an example of that in action.